### PR TITLE
newwww

### PR DESCRIPTION
**Pull request by sprocivjs**
newww

### CODE REVIEW

**Comment by DemchyshynV** [comment]
hellp

**Comment by sprocivjs** [comment]
just to do it

**Comment by sprocivjs** [comment]
how are you

**Comment by sprocivjs** [comment]
litle bit

**Comment by sprocivjs** [comment]
what??

**Comment by DemchyshynV** [comment]
hhhhh

**Comment by sprocivjs** [comment]
jay

File: bewwwwwww.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+
```
Comment (DemchyshynV) [inline bewwwwwww.html:8]: А где код?

**Review by DemchyshynV — COMMENTED** [review]
ываыва

File: bewwwwwww.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
```
Comment (sprocivjs) [inline bewwwwwww.html:7]: okayyy

**Review by DemchyshynV — APPROVED** [review]

**Pull request merged**